### DISCUSSION TITLE
0주차 미팅

### DISCUSSION THREAD

**Discussion by nyj001012** [GunGonGamLee/ft_transcendence#1] [General]
# 논의할 내용
- 협업 방식
- 어떤 기능을 구현할 것인가?
- 일정 협의
- 기술 스택

**Comment by nyj001012** [comment]
## 논의 내용
### 협업 방식
- 오프라인 미팅
- 코어타임: 평일 13시 - 18시

### 어떤 기능을 구현할 것인가?
| 타입 | 내용 | 주요 / 부수 |
|:---:|:-----|:---------:|
| 웹 | 백엔드로 프레임워크 사용. | 주요 |
| 웹 | 백엔드용 데이터베이스 사용. | 부수 |
| 사용자 관리 | 표준 사용자 관리, 인증, 토너먼트 간 사용자.  | 주요 |
| 사용자 관리 | 원격 인증 구현. | 주요 |
| 게임플레이 및 사용자 경험 | 원격 플레이어. | 주요 |
| 게임플레이 및 사용자 경험 | 멀티플레이어(동일 게임 내 2명 이상). | 주요 |
|  AI - 알고리즘 | 사용자 및 게임 통계 대시보드. | 부수 |
| 사이버 보안 | WAF/ModSecurity 및 HashiCorp Vault를 이용한 강화된 설정과 비밀 관리 구현. | 주요 |
| 사이버 보안 | 2단계 인증(2FA) 및 JWT 구현. | 주요 |
| 데브옵스 | 로그 관리를 위한 인프라 설정. | 주요 |
| 데브옵스 | 백엔드를 마이크로서비스로 설계. | 주요 |
| 접근성 | 다중 언어 지원. | 부수 |

- 주요: 9
- 부수: 2

### 일정 협의
- 마감 기한: 2월 내

### 기술 스택
#### 프론트
#### 백엔드
- Django
- PostgreSQL
- Redis
- Elasticsearch
- Logstash
- Kibana

#### 공용
- Swagger

**Comment by sejoonkimmm** [comment]
* 일정 중 빠지는 날이 있으면 미리 말씀해주세요! (ex. sejokim은 마스터 회의 등...) 아픈데 무리해서 나오는 일은 없도록 해요...!
